application/src/main/java/org/thingsboard/server/service/security/auth/jwt/settings/DefaultJwtSettingsService.java

```diff
@@ -124,6 +128,7 @@ public JwtSettings getJwtSettings(boolean forceReload) {
                         log.warn("WARNING: The platform is configured to use default JWT Signing Key. " +
                                 "This is a security issue that needs to be resolved. Please change the JWT Signing Key using the Web UI. " +
                                 "Navigate to \"System settings -> Security settings\" while logged in as a System Administrator.");
+                        notificationCenter.sendGeneralWebNotification(TenantId.SYS_TENANT_ID, new SystemAdministratorsFilter(), DefaultNotifications.jwtSigningKeyIssue.toTemplate());
```

Comment:
For me, it is better that JwtSettingsService know nothing about how to compose the notification.

It is much clear to have a simple call like 'notificationCenter.notifyJwtSettingsKeyIssue()'

Reply:
JwtSettingsService only chooses needed notification from DefaultNotifications.
I don't want NotificationCenter to end up looking like MailService :)